src/pyramid/config/routes.py

```diff
@@ -220,25 +220,23 @@ def add_route(
         header
 
           This value can be a string or an iterable of strings for HTTP
-          header names.  Any string that does not contain a ``:``
-          (colon) will be considered to be the header name (e.g.
-          ``If-Modified-Since``).  In this case, the header specified by
-          the name must be present in the request for this predicate
-          to be true.
-
-          If a string contains a colon, it will be considered a
-          name/value pair (e.g. ``User-Agent:Mozilla/.*`` or
-          ``Host:localhost``), where the value part is a regular
-          expression.  The header specified by the name must be present in
-          the request *and* the regular expression specified as the
-          value must match the header value.
-
-          Whether or not the strings
-          represent a header name or a header name/value pair, the
-          case of the header name is not significant.  If this
-          predicate returns ``False``, route matching continues.
-          When the value is an iterable of strings, they must all
-          return ``True`` for this predicate to return ``True``.
+          header names.  The header names are determined as follow:
+
+          - If a string does not contain a ``:`` (colon), it will be
+            considered to be the header name (e.g.  ``If-Modified-Since``).
```

Comment:
I much prefer "For example" because I never pronounce "e.g." when reading or speaking.

Nonetheless, if it is to be used, then an extra space is not needed.

```suggestion
            considered to be the header name (e.g. ``If-Modified-Since``).
```